README.md

```diff
@@ -55,50 +55,47 @@ The source code in this package is released under the Apache-2.0 License. For fu
 The [3rdparty-licenses.txt](3rd-party-licenses.txt) contains a list of other open source components included in this package.
 
 
-## Installation & Usage as a ROS package
-
 ### Dependencies
 This package depends on the linear algebra library [Eigen], which can be installed by
 
     sudo apt-get install libeigen3-dev
 
-The [ROS] dependencies are listed in the package.xml and can be installed by
+The [ROS 1] or [ROS 2] dependencies are listed in the package.xml and can be installed by
 
     rosdep install steering_functions
 
 
 ### Building
 
-To build this package from source, clone it into your catkin workspace and compile it in *Release* mode according to
+To build this package from source, clone it into your workspace and compile it in *Release* mode according to
 
-    cd catkin_ws/src
+    cd ws/src
     git clone https://github.com/hbanzhaf/steering_functions.git
-    catkin build steering_functions -DCMAKE_BUILD_TYPE=Release
+    catkin build steering_functions -DCMAKE_BUILD_TYPE=Release  # [ROS 1]
```

Comment:
Why do we need this line when the next line is for ROS 1 and ROS 2?

Reply:
Most ROS 1 user will use `catkin` for compilation. But colcon can be used for ROS 1 as well. So I think it's most clear to list them both and users can choose.

Reply:
Okay, then let's just have '# ROS 1' (without the square brackets) and in the following line '# ROS 1 or ROS 2'